### PR TITLE
feat(content): Thick socks

### PR DESCRIPTION
**Pull request by yay855**
## Purpose of change

Normal socks are not very warm, but wool socks can be annoying to acquire. Thicker cotton socks makes perfect sense.

## Describe the solution

Added in new item, thick socks, and corresponding recipe. They're half as warm as wool socks.

### CODE REVIEW

File: data/json/items/armor/boots.json [hunk 1]
```diff
@@ -1455,5 +1455,14 @@
...
     "flags": [ "VARSIZE", "WATER_FRIENDLY", "STURDY", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "socks_thick",
+    "type": "ARMOR",
+    "name": { "str": "pair of thick socks", "str_pl": "pairs of thick socks" },
+    "copy-from": "socks",
+    "looks_like": "socks",
+    "warmth": 15,
+    "material_thickness": 2
```
Comment (chaosvolt) [inline data/json/items/armor/boots.json:1466]: 5 more warmth than stockings but still zero encumbrance? o3o

Comment (scarf005) [reply data/json/items/armor/boots.json:1466]: yeah, increasing encumbrance a bit could work.

**Review by RoyalFox2140 — CHANGES_REQUESTED** [review]
I own a pair of something like this IRL and I can say its at least 2 or 3 encumbrance because its slightly harder to walk around in. The extra material is a bit cumbersome but not by that much. I'd also say for balance we tend to have winter clothing give higher encumbrance, especially because of layering blunt and cut armor in winter to fight zombies without taking damage, so it needs a trade-off so it would follow more internal balance consistency to encumber a player.
image

File: data/json/items/armor/boots.json (hunk 1 above)
Comment (yay855) [reply data/json/items/armor/boots.json]: There we go, added 3 encumbrance to them.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**